### PR TITLE
feat: product listing

### LINKED ISSUE #2: Product Listing Page
### Product Listing Page
- [ ] Product grid
- [ ] Filter sidebar
- [ ] Filter by price
- [ ] Filter by category
- [ ] Filter by ratings

### PR DESCRIPTION
**Pull request by siddhigate**
### Product listing page
- Products grid
- Filters by price, ratings, categories, brands

**Live Preview:** https://fashionly-ui-git-productl-listing-page-siddhigate.vercel.app/pages/productlisting.html

productlist

### CODE REVIEW

File: css/pages/productlisting.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+
+@media screen and (max-width: 1200px) {
+  .product-listing {
+    width: 80%;
+  }
+}
+
+@media screen and (max-width: 1200px) {
```
Comment (adidoshi) [inline css/pages/productlisting.css]: Maybe you can put this code in one media query as the `max-width` is same for both.

**Review by gopalbbharadva — APPROVED** [review]
Nice UI:

**Review by adidoshi — APPROVED** [review]
The UI looks great! 👍
Just one suggestion, I see that the filter sidebar gets scrolled when we are at the end of the page, if you implemented that way, good or maybe you can check once.

File: css/pages/productlisting.css (hunk 1 above)
Comment (siddhigate) [reply css/pages/productlisting.css]: Thanks for pointing these issues! Have solved those

**Pull request merged**